node/speech-recognition-with-huggingface/README.md

```diff
@@ -0,0 +1,105 @@
+# Speech Recognition with Hugging Face
+
+This function uses the Hugging Face API to perform speech recognition. It takes an flac file from Appwrite storage and sends it to the Hugging Face API for speech recognition. The API returns the text and records it in the database. This function also supports receiving document events from the Appwrite Database.
```

Comment:
does it have to be a flac file or can also be a wav? whats the most popular file type, I think we should support that 🤔